src/mixins/Projector.ts

```diff
@@ -1,15 +1,14 @@
 import { assign } from '@dojo/core/lang';
-import global from '@dojo/shim/global';
 import { createHandle } from '@dojo/core/lang';
 import { Handle } from '@dojo/core/interfaces';
-import 'pepjs';
 import cssTransitions from '../animations/cssTransitions';
 import { Constructor, DNode, Projection, ProjectionOptions } from './../interfaces';
 import { WidgetBase } from './../WidgetBase';
 import { afterRender } from './../decorators/afterRender';
 import { v } from './../d';
 import { Registry } from './../Registry';
-import { dom, widgetInstanceMap } from './../vdom';
+import { dom } from './../vdom';
+import 'pepjs';
```

Comment:
I believe this import can be removed as it's handled in [shim](https://github.com/dojo/shim/blob/f2594e2d6f91865464d8a10ba80b76e1aa4e16f1/src/browser.ts#L1)